build/build.umple.xml

```diff
@@ -40,6 +41,12 @@
     <delete dir="cruise.umple.validator/src-gen-umple" failonerror="false" />
   </target>
 
+  <target name="init">
+    <antcall target="template.init" />
+    <!-- TODO: get the "-f build.deps.xml bootstrap" to be part of the docs / build and then remove this line -->
+    <ant antfile="build/build.deps.xml" target="deps.vendor" inheritAll="true" />

```

Comment:
`bootstrap` can only be included *iff* we remove all references to Ant Contrib in the `first-build` step.